xbox/webapi/api/provider/people/__init__.py

```diff
@@ -0,0 +1,162 @@
+"""
+People - Access friendlist from own profiles and others
+"""
+from typing import List
+
+from xbox.webapi.api.provider.baseprovider import BaseProvider
+from xbox.webapi.api.provider.people.models import (
+    PeopleDecoration,
+    PeopleResponse,
+    PeopleSummaryResponse,
+)
+
+
+class PeopleProvider(BaseProvider):
+    SOCIAL_URL = "https://social.xboxlive.com"
+    HEADERS_SOCIAL = {"x-xbl-contract-version": "2"}
+    PEOPLE_URL = "https://peoplehub.xboxlive.com"
+    HEADERS_PEOPLE = {
+        "x-xbl-contract-version": "3",
+        "Accept-Language": "overwrite in __init__",
+    }
+    SEPERATOR = ","
+
+    def __init__(self, client):
+        """
+        Initialize Baseclass, set 'Accept-Language' header from client instance
+
+        Args:
+            client (:class:`XboxLiveClient`): Instance of client
+        """
+        super().__init__(client)
+        self.HEADERS_PEOPLE.update({"Accept-Language": self.client.language.locale})
```

Comment:
This will update not only the instance attribute but also the class attribute. Is that intended? The attribute is defined as a constant with uppercase letters, so it's not expected to change.

```
Python 3.8.5 (default, Jul 28 2020, 12:59:40) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.18.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: class PeopleProvider:
   ...:     HEADERS_PEOPLE = {
   ...:         "x-xbl-contract-version": "3",
   ...:         "Accept-Language": "overwrite in __init__",
   ...:     }
   ...: 
   ...:     def __init__(self, locale):
   ...:         self.HEADERS_PEOPLE.update({"Accept-Language": locale})
   ...: 

In [2]: PeopleProvider.HEADERS_PEOPLE
Out[2]: {'x-xbl-contract-version': '3', 'Accept-Language': 'overwrite in __init__'}

In [3]: provider = PeopleProvider("us")

In [4]: provider.HEADERS_PEOPLE
Out[4]: {'x-xbl-contract-version': '3', 'Accept-Language': 'us'}

In [5]: PeopleProvider.HEADERS_PEOPLE
Out[5]: {'x-xbl-contract-version': '3', 'Accept-Language': 'us'}
```

Reply:
That's fair. This is actually taking a page from some of the other providers. I'll update them all to use instance headers.